sqlmesh/core/scheduler.py

```diff
@@ -39,17 +39,17 @@
 from sqlmesh.utils.errors import AuditError, CircuitBreakerError, SQLMeshError
 
 logger = logging.getLogger(__name__)
-Interval = t.Tuple[datetime, datetime]
-Batch = t.List[Interval]
-SnapshotToBatches = t.Dict[Snapshot, Batch]
+Interval = t.Tuple[int, int]
```

Comment:
this should continue to be datetime because users implement check_intervals method